company.js

```diff
@@ -0,0 +1,86 @@
+let loader2=document.querySelector(`.spinner-border2`);
+let container= document.querySelector(`.company-container`);
+let container2= document.querySelector(`.company-container2`);
+var urlParams = new URLSearchParams(window.location.search);
+var extractedSymbol=urlParams.get(`symbol`)
+let prices=[]
+let dates=[]
+
+ async function getSymbol(){
```

Comment:
you alredy have a code that sends request to the serverin order to get the company profile. you should make sure you are using this code/function agian, instead of writing it agian.